modules/core/src/test/java/org/apache/ignite/testsuites/IgniteCacheTestSuite.java

```diff
@@ -215,6 +216,8 @@
         GridTestUtils.addTestIfNeeded(suite, IgniteCachePrivateExecutionContextTest.class, ignoredTests);
         GridTestUtils.addTestIfNeeded(suite, IgniteCacheSharedExecutionContextTest.class, ignoredTests);
 
+        GridTestUtils.addTestIfNeeded(suite, StoreArrayKeyTest.class, ignoredTests);
```

Comment:
Why was addTestIfNeeded used? In which cases should it be filtered out?

Reply:
I used it because other tests in this class use it.